### PR TITLE
Fix docs of options argument in RawKernel and RawModule

### PR DESCRIPTION
**Pull request by kmaehashi**

### CODE REVIEW

File: cupy/core/raw.pyx [hunk 1]
```diff
@@ -216,9 +218,11 @@ cdef class RawModule:
...
     Args:
         code_or_path (str): CUDA source code or path to cubin.
-        options (str): Compiler options passed to NVRTC if compilation is
-            needed. For details, see
-            https://docs.nvidia.com/cuda/nvrtc/index.html#group__options.
+        options (tuple of str): Compiler options passed to the backend (NVRTC
+            or NVCC). For details, see
+            https://docs.nvidia.com/cuda/nvrtc/index.html#group__options or
+            https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#command-option-description
+        backend (str): Either `nvrtc` or `nvcc`. Defaults to `nvrtc`
```
Comment (leofang) [inline cupy/core/raw.pyx]: This line is a duplicate?

Comment (kmaehashi) [reply cupy/core/raw.pyx]: Good catch!

**Comment by asi1024** [comment]
Jenkins, test this please.

**Comment by asi1024** [comment]
Jenkins, test this please.

**Comment by asi1024** [comment]
LGTM.

**Pull request merged**